config/astral.yml

```diff
@@ -18,10 +18,25 @@ shared:
 
   audit_log_file: <%= "#{Rails.root.join('log')}/astral-audit.log" %>
 
+  oidc_issuer:
+  oidc_client_id:
+  oidc_client_secret:
+  oidc_redirect_uris: http://localhost:8250/oidc/callback
+
 test:
+  oidc_issuer: http://oidc_provider:8300/v1/identity/oidc/provider/astral
+  oidc_provider_addr: http://oidc_provider:8300
+  initial_user_name: test
+  initial_user_password: test
+  initial_user_email: test@example.com
```

Comment:
you could eliminate one repetition of this config by putting these dev/test values in `shared` and the blank overrides in `production`

Reply:
done